apps/src/aichat/redux/aichatRedux.ts

```diff
@@ -296,6 +297,14 @@ export const submitChatContents = createAsyncThunk(
       aichatContext,
       currentSessionId
     );
+    Lab2Registry.getInstance()
+      .getMetricsReporter()
+      .reportLoadTime('AichatModelResponseTime', Date.now() - startTime, [
+        {
+          name: 'Pathname',
+          value: window.location.pathname,
+        },
+      ]);
```

Comment:
This is question is out of scope for the PR, but related to the dimensions being reported and directed at @sanchitmalhotra126 . I see that the common dimension  `AppName` is included in the reporting. But curious why the other `commonProperties` (e.g., `channelId`, `currentLevelId`,...) are not included when calling [on `getCommonDimensions`](https://github.com/code-dot-org/code-dot-org/blob/be7f35b66d20a6cf09dbc66b00daff61b3868520/apps/src/lab2/Lab2MetricsReporter.ts#L89).

Reply:
Yep, added some clarifications around metric dimensions in a separate comment. I think we probably don't want to include Pathname as a dimension since we don't want to emit distinct metrics for each level, but it would be good to include the selected model ID as a dimensions so we can view those metrics separately? Trying to think if there's any other fields we want to similarly bucketize by, but that's the first that comes to mind.

Reply:
This makes sense - thanks for the explanations!